docs/source/index.rst

```diff
@@ -22,7 +22,7 @@ Installing ``poke-env``
 
 Installation can be performed via pip with:
 
-.. code-block::
+.. code-block:: bash
```

Comment:
By curiosity, do you know why this is necessary? According to #1 building locally totally works.